<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthenticateWithTwitchDialog</class>
 <widget class="QDialog" name="AuthenticateWithTwitchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuthenticateWithTwitch</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="authenticateInBrowserButton">
     <property name="text">
      <string>AuthenticateInBrowser</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="authenticateWithAccessTokenGroup">
     <property name="title">
      <string>OrAuthenticateWithAccessToken</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="accessTokenEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="authenticateWithAccessTokenButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
